tests/test.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+echo "$(date)"
+
+readonly ROOT_DIR="$(dirname "$(dirname "${0}")")"
+source "${ROOT_DIR}"/tests/functions.sh
+
+readonly IMG=""
+readonly NAMESPACE="sumologic-system"
+readonly TIME=900
+
+wget https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/main/vagrant/k8s/receiver-mock.yaml
+kubectl apply -f receiver-mock.yaml
+
+helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
+helm repo update
+
+helm install test-openshift sumologic/sumologic \
+  --set sumologic.accessId="dummy" \
+  --set sumologic.accessKey="dummy" \
+  --set sumologic.endpoint="http://receiver-mock.receiver-mock:3000/terraform/api/" \
+  --set sumologic.scc.create=true \
+  --set fluent-bit.securityContext.privileged=true \
+  --set kube-prometheus-stack.prometheus-node-exporter.service.port=9200 \
+  --set kube-prometheus-stack.prometheus-node-exporter.service.targetPort=9200 \
+  --set fluentd.buffer.queueChunkLimitSize=256 \
+  --set fluentd.buffer.totalLimitSize="256m" \
+  --set metrics-server.enabled=true \
+  --set metrics-server.apiService.create=false \
+  --set otelagent.enabled=true \
+  --set telegraf-operator.enabled=true \
+  --set falco.enabled=true \
+  --set tailing-sidecar-operator.enabled=true \
+  --version 2.1.0 \
+  -n sumologic-system \
+  --create-namespace
+
+wait_for_resource ${NAMESPACE} ${TIME} daemonset.apps/test-openshift-falco
```

Comment:
you can use [kubectl wait](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait) to wait for resources to be in healthy state

Reply:
`kubectl wait` may wait for resource available on api, I've noticed that sometimes resources are not available immediately and this caused break of tests but resources were created a little bit later... I had make such changes in tailing-sidecar: https://github.com/SumoLogic/tailing-sidecar/commit/cd72d3e33887bebdd8bb1c856f92a43014a65e25